src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/config/HmppsTemplateKotlinExceptionHandler.kt

```diff
@@ -35,6 +36,17 @@ class HmppsTemplateKotlinExceptionHandler {
       ),
     ).also { log.info("No resource found exception: {}", e.message) }
 
+  @ExceptionHandler(AccessDeniedException::class)
+  fun handleAccessDeniedException(e: AccessDeniedException): ResponseEntity<uk.gov.justice.hmpps.kotlin.common.ErrorResponse> = ResponseEntity
```

Comment:
Not related to this PR, just something to ponder - would we ever add an exception handler to the library? I think the ErrorResponse is already in there.

Reply:
I _think_ we could add some default advice with
```
@Order(Ordered.LOWEST_PRECEDENCE)
``` that would then be overridden by any other advice.  Therefore sounds a simple case of just adding the handler to the library.